Makefile

```diff
@@ -58,19 +57,17 @@ busybox.tgz:
 	docker save busybox:latest | gzip -c > busybox.tgz
 
 calico-node.tgz:
-	docker pull calico/node:latest
-	docker save calico/node:latest | gzip -c > calico-node.tgz
+	docker pull calico/node:v0.10.0
```

Comment:
The code under test is the libnetwork-plugin code - it doesn't matter too much what version of the node is used. It needs to match the version that calicoctl expects.